R/app_server.R

```diff
@@ -171,6 +171,25 @@ app_server <- function(input, output, session) {
   }) %>%
     bindEvent(selected_pkg$decision(), selected_pkg$overall_comment_added())
   
+  
+  rpt_choices <- c(
+    "Report Author", "Report Date", "Risk Score", "Overall Comment", 
+    "Package Summary", "Maintenance Metrics", "Maintenance Comments", 
+    "Community Usage Metrics", "Community Usage Comments", "Source Explorer Comments"
+    )
```

Comment:
I don't think we want this value populated in `app_server.R` and `mod_downloadHandler.R` especially since it will be changing as we add more to the app. I see two possible solutions:

1. We store it as an internal data value.
2. We specify it in `app_server.R` and add another value to `session$userData$user_report`.

Personally I prefer approach one. With approach one we could potentially write tests to make sure that it is getting updated with changes in the app too.

Reply:
copied rpt_choices to `internal-data.R` and removed from `app_server.R` and `mod_downloadHanlder.R`